addons/vehicles/config.cpp

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+
+#include "CfgVehicles.inc"
+
+class CfgPatches {
+    class ADDON {
+        name = QUOTE(COMPONENT);
+        units[] = { CLASSES_VEHICLES };
+        weapons[] = { };
+        magazines[] = { };
+        ammo[] = { };
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = { 
+            "SWS_Main",
+            "SWS_Weapons",
+            "A3_Armor_F_EPB",
+            "ace_medical"
```

Comment:
```suggestion
            "ace_medical",
            "OPTRE_UNSC_Units_Army",
            "B_19thUNSC"
```

You use the OPTRE soldiers for the crew and 19th for the cargo so they need to be dependencies.